extra_playbooks/build-rbd-provisioner.yml

```diff
@@ -0,0 +1,55 @@
+---
+
+- hosts: localhost
+  tasks:
+    - name: RBD Provisioner | Install pip packages
+      pip:
+        name: "{{ item.name }}"
+        version: "{{ item.version }}"
+        state: "{{ item.state }}"
+        extra_args: "{{ pip_extra_args | default(omit) }}"
+      with_items:
+        - { state: "present", name: "docker", version: "3.4.1" }
```

Comment:
Fixed versions here seems unlogical

Reply:
This Playbook is used to build provisioner images only and is not necessary during deployment process even though RBD provisioner is enabled.

This is based on current CephFS provisioner (https://github.com/kubernetes-sigs/kubespray/blob/master/extra_playbooks/build-cephfs-provisioner.yml). If not essential, I will remove it.